docs/api/features/column-sizing.md

```diff
@@ -152,6 +152,14 @@ columnResizeMode?: 'onChange' | 'onEnd'
 
 Determines when the columnSizing state is updated. `onChange` updates the state when the user is dragging the resize handle. `onEnd` updates the state when the user releases the resize handle.
 
+### `columnResizeDirection`
```

Comment:
isnt there a global setting to rtl, that we can depend on instead of allowing rtl specifically for column resizing?

Reply:
In another PR, I have introduced a global option rtl that could be used in other places.

But, this is the only feature that requires an explicit rtl setting. Other things can be sorted using CSS properties.

Therefore, I thought it would be more appropriate to make this a feature-specific option.